cmake/Platform/Common/Clang/Configurations_clang.cmake

```diff
@@ -54,6 +54,7 @@ ly_append_configurations_options(
         -fstack-check
 
     COMPILATION_RELEASE
+        -g
```

Comment:
This needs to be removed.
Release shouldn't have debug information in it

Reply:
Will fix.